src/FamilyHubs.Referral.Core/AutoMappingProfiles.cs

```diff
@@ -12,33 +12,47 @@ public AutoMappingProfiles()
             .ForMember(dest => dest.Created, opt => opt.Ignore())
             .ForMember(dest => dest.LastModified, opt => opt.Ignore())
             .ForMember(dest => dest.Status, opt => opt.MapFrom(src => src.Status))
-            .ForMember(dest => dest.ReferralUserAccount, opt => opt.MapFrom(src => src.ReferrerDto))
+            .ForMember(dest => dest.ReferralUserAccount, opt => opt.MapFrom(src => src.ReferralUserAccountDto))
```

Comment:
Is this the right dto? We shouldn't have a ReferralUserAccounts table now, the the foreign key from the UserAccount table.

Reply:
Dto has that name would need to upgrade the NuGet Package